.github/workflows/comment_to_issue.yml

```diff
@@ -0,0 +1,20 @@
+name: Comment to Issue
+on: [push, pull_request]
```

Comment:
is there a way to trigger the action only after merging a PR and only for the `main` branch? we will be deleting feature branches and the issues are currently referring to the branch that created the PR

Reply:
@giannou I changed the code according to your request. From now on, the action will only be triggered after a PR is closed and merged successfully to `main`. Also, I have added a new `FIXME:` comment to test the new condition. If everything goes well, the issue will be created after the PR is merged.